src/browser/replay/recorder.js

```diff
@@ -4,15 +4,24 @@ import { EventType } from '@rrweb/types';
 import hrtime from '../../tracing/hrtime.js';
 import logger from '../../logger.js';
 
+/** @typedef {import('./recorder.js').BufferCursor} BufferCursor */
+
 export default class Recorder {
   _options;
   _rrwebOptions;
+
+  _isReady = false;
```

Comment:
Moved `_isReady` initialization here from the constructor to follow the pattern we use with the rest of the initialized-at-instancing properties.